packages/arb-token-bridge-ui/src/components/MainContent/ExploreArbitrumContent.ts

```diff
@@ -90,7 +90,8 @@ export const ExploreArbitrumContent = {
     },
     {
       name: 'KyberSwap',
-      description: 'Multichain Aggregator with efficient swap rates & yield farming on Arbitrum, code audited. Free pro trading tools.',
+      description:
```

Comment:
Why this line-break change?

Reply:
Prettier. If I run prettier on this file, then I got the same diff.

Reply:
yep, prettier change. It was always coming up in yarn format so decided to finally let it :D

Reply:
Yes, it better be automated :)